clamav.py

```diff
@@ -194,7 +194,7 @@ def scan_file(path):
         stdout=subprocess.PIPE,
         env=av_env,
     )
-    output = av_proc.communicate()[0]
+    output = av_proc.communicate()[0].decode(encoding='UTF-8', errors='ignore')
```

Comment:
So this actually one of the differences between python 2 and 3. Python 2 `output` is a string, where in python 3, it's bytes - that why we have to decode.

Just ignoring errors seems to work, there is some garbage that get's produced in the logs/output in case files are non UTF-8, but that's roughly equivalent to python 2 version which also produces garbage in the logs in case of non UTF-8

Reply:
Nice work!